lcls-twincat-pmps/PMPS/HelperFunctions/F_SafeBPCompare.TcPOU

```diff
@@ -20,18 +20,26 @@ VAR
     
     idx : UINT;
     xAuxAttOK : BOOL := TRUE;
-    xAuxAprtOK : BOOL := TRUE; 
+    xAuxAprtOK : BOOL := TRUE;
+    
+    Att1 : REFERENCE TO ST_PMPS_Attenuator;
+    Att2 : REFERENCE TO ST_PMPS_Attenuator;
+    
 END_VAR
 ]]></Declaration>
     <Implementation>
       <ST><![CDATA[
-xAttOK := BeamParam1.nTran <= BeamParam2.nTran;
+xAttOK := BeamParam1.nTran <= (BeamParam2.nTran *(100+PMPS_PARAM.ATT_MARGIN))/PMPS_GVL.TRANS_SCALING_FACTOR;
```

Comment:
Will this break PMPS requests for nTran = 100? It seems like those would require the arbiter to claim 105% transmission

Reply:
I agree, this doesn't make much sense so I'll add some bounds.